README.md

```diff
@@ -1,4 +1,22 @@
-## Inventory
+# Inventory
+
+Demo app for showing how Ditto sync and counters work.
+
+This inventory demo showcases the smoothness of the Ditto sync and how counters work with Ditto. You can open up the Presence Viewer to see the Ditto connections too.
+
+Powered by [Ditto](https://www.ditto.live/).
+
+- [Demo Video](https://www.youtube.com/watch?v=1P2bKEJjdec)
+
+- [iOS Download](https://apps.apple.com/us/app/ditto-inventory/id1449905935)
+- [Android Download](https://play.google.com/store/apps/details?id=live.ditto.inventory)
+
+
+## Developing
+
+If you are a member of Ditto, please access to this page and copy-paste the environment variables to this project in order to build the app:
```

Comment:
"If you work for Ditto, please visit this page..."

Reply:
We should move this out into a Notion page to reduce noise